package.json

```diff
@@ -24,36 +24,33 @@
   },
   "dependencies": {
     "@bugsnag/js": "6.5.2",
-    "@bugsnag/plugin-express": "6.5.1",
-    "@nestjs/common": "6.11.11",
-    "@nestjs/core": "6.11.11",
-    "@nestjs/microservices": "6.11.11",
-    "@nestjs/platform-express": "6.11.11",
-    "@nestjs/testing": "6.11.11",
-    "@nestjs/websockets": "6.11.11",
+    "@bugsnag/plugin-express": "6.4.1",
```

Comment:
why to shift bugsnag down?

Reply:
Reverted this change.. I tried to upgrade it, but there were many breaking changes that I didn't want to get into, so I downgraded it to the wrong version

Reply:
we actually have a ticket for it